### ISSUE TITLE
Allow getFileFromManifest to load files from webpack-dev-server

### ISSUE THREAD

**Issue by Kashkin** [nystudio107/craft-twigpack#23]
I've taken at how how I think this function should work, but I have two main concerns:

1. It works for my setup, where if I've run a build, this will return built and cache-broken local files. And, if I'm running webpack-dev-server, that script will first clean my built files so that the logic of this function works.
2. I don't think the config variables I'm using are the best choice, but again, they work for me.

```php
public static function getFileFromManifest(array $config, string $fileName, string $type = 'legacy'): string
{
	$path = null;
	try {
		$path = self::getModuleEntry($config, $fileName, $type, true);
	} catch (NotFoundHttpException $e) {
		Craft::error($e->getMessage(), __METHOD__);
	}
	if ($path !== null) {
		// Let's try and get a built file first
		$localPrefix = $config['localFiles']['basePath'] . $config['localFiles']['criticalPrefix'];
		$localPath = self::combinePaths(
			$localPrefix,
			$path
		);
		$alias = Craft::getAlias($localPath, false);
		if ($alias && is_string($alias)) {
			$localPath = $alias;
		}
		if (is_file($localPath)) {
			return self::getFile($localPath) ?? '';
		}
		// If that failed, maybe it's available from the webpack-dev-server
		$devServerPrefix = $config['devServer']['publicPath'];
		$devServerPath = self::combinePaths(
			$devServerPrefix,
			$path
		);
		return self::getFileFromUri($devServerPath, null) ?? '';
	}
	return '';
}

**Comment by khalwat** [comment]
So this probably would work; but the logic that Twigpack follows for most things is to try the devServer first, and if that fails, then fall back on trying the local version.

Do you want to do up a PR?

**Comment by khalwat** [comment]
Fixed in 1.1.9 via your PR. Thanks!
